assets/src/application/ui/layout/entityList/EntityList/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { without } from 'ramda';
+import classNames from 'classnames';
+
+import { Entity } from '@edtrServices/apollo/types';
+import EmptyState from '@appDisplay/EmptyState';
+import './style.scss';
+
+interface EntityListProps {
+	entities: Entity[];
+	EntityGridView: React.ReactNode;
+	EntityListView: React.ReactNode;
+	htmlClass: string;
```

Comment:
I think we should decide about following a standard whether we want to use `htmlClass` or `className` or `classNames`.

Reply:
agree, let's loop back on this after we will have that decision consolidated

Reply:
just discussed in chat that we will switch to using `className` but those can all be fixed in a later PR where we go through and polish things at the end